spec/models/work_statistic_spec.rb

```diff
@@ -316,15 +315,7 @@
       end
       it 'recalculates with all page status types' do
         # Ensures compatibility with old API
-        statuses = %w[
-          transcribed
-          corrected
-          translated
-          blank
-          indexed
-          review
-        ]
-        statuses.each do |status|
+        Page.statuses.each_value do |status|
```

Comment:
Does `statuses` produce a hash instead of an array/list?  (Wondering why it's necessary to call `each_value` instead of `each` here.)

Reply:
Yes, it produces a hash. We can do `Page.statuses.each do |key, value|` but it's essentially the same with each_value